dashed/assets/javascripts/welcome.js

```diff
@@ -0,0 +1,53 @@
+var $ = window.$ = require('jquery');
+var jQuery = window.jQuery = $;
+var px = require('./modules/dashed.js');
+
+require('../stylesheets/dashed.css');
+require('../stylesheets/welcome.css');
+
+require('bootstrap');
+require('datatables');
+require('d3');
+
+require('../node_modules/cal-heatmap/cal-heatmap.css');
+var CalHeatMap = require('cal-heatmap');
+
+
+
+function modelViewTable(selector, modelEndpoint, ordering) {
+  // Builds a dataTable from a flask appbuilder api endpoint
+  $.getJSON(modelEndpoint + '/api/read', function (data) {
+    var tableData = jQuery.map(data.result, function(el, i) {
+        var row = $.map(data.list_columns, function(col, i) {
+          return el[col];
+        });
+        return [row];
+    });
+    var cols = jQuery.map(data.list_columns, function(col, i) {
+      return { "sTitle": data.label_columns[col] }
+    });
+    $(selector).DataTable({
+      aaData: tableData,
+      aoColumns: cols,
+      bPaginate: false,
+      order: ordering,
+      searching: false
+    });
+    $('[data-toggle="tooltip"]').tooltip({ container: 'body' });
+  });
+}
+
+$(document).ready(function () {
+  var cal = new CalHeatMap();
+  cal.init({
+    start: new Date().setFullYear(new Date().getFullYear() - 1),
+    range: 13,
+    data: '/dashed/activity_per_day',
+    domain : "month",
+    subDomain : "day",
+    itemName: "action",
+    tooltip: true
+  });
+  modelViewTable('#dash_table', '/dashboardmodelviewasync');
```

Comment:
just something to note: every time you give a DOM element an `id`, it creates a GLOBAL javascript variable with that id name. there are tons of instances of using `#my_id` selectors in the code base so it's probably not worth changing at this point, but going forward it is better to use `class`es than `id`s for DOM elements and selections.